src/mods.cpp

```diff
@@ -28,11 +28,44 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "util/strfnd.h"
 #include "convert_json.h"
 #include "exceptions.h"
+#include <vector>
+
+static void independentGetLine(std::istream& is, std::string& line)
+{
+    // The characters in the stream are read one-by-one using a std::streambuf.
```

Comment:
@est31 This could just be removed entirely. CR is probably not going to be found anyway and the current upstream (std::getline() + trim()) gets rid of the CR on CRLFs on systems which only strip the LF with std::getline.